### ISSUE TITLE
Recommendation: I used conda for training.

### ISSUE THREAD

**Issue by Sunshine8921** [ai4ce/GARF#17]
I hope this helps you.
accelerate                1.0.1
addict                    2.4.0
aiofiles                  23.2.1
aiohappyeyeballs          2.4.4
aiohttp                   3.10.11
aiosignal                 1.3.1
annotated-types           0.7.0
antlr4-python3-runtime    4.9.3
anyio                     4.5.2
appdirs                   1.4.4
asttokens                 3.0.0
async-timeout             5.0.1
attrs                     25.3.0
backcall                  0.2.0
bayesian-optimization     1.4.3
blinker                   1.8.2
ccimport                  0.4.4
certifi                   2022.12.7
cffi                      1.17.1
cfgv                      3.4.0
chamfer                   2.0.0
chamferdist               1.0.3
charset-normalizer        2.1.1
click                     8.1.8
cmake                     3.25.0
colorama                  0.4.6
comm                      0.2.3
ConfigArgParse            1.7.1
contourpy                 1.1.1
cumm-cu118                0.4.11
cycler                    0.12.1
dash                      3.2.0
decorator                 5.2.1
diffusers                 0.30.0
distlib                   0.4.0
docker-pycreds            0.4.0
einops                    0.8.1
exceptiongroup            1.3.0
executing                 2.2.0
fastapi                   0.116.2
fastjsonschema            2.21.1
ffmpy                     0.5.0
filelock                  3.13.1
fire                      0.7.1
flash-attn                2.5.2
Flask                     3.0.3
fonttools                 4.57.0
frozenlist                1.5.0
fsspec                    2024.12.0
future                    0.18.3
garf                      0.0.2
gitdb                     4.0.12
GitPython                 3.1.45
gradio                    4.44.1
gradio_client             1.3.0
h11                       0.16.0
h5py                      3.11.0
hf-xet                    1.1.5
httpcore                  1.0.9
httpx                     0.28.1
huggingface-hub           0.25.2
hydra-core                1.3.2
identify                  2.6.1
idna                      3.4
imageio                   2.31.6
importlib_metadata        8.5.0
importlib_resources       6.4.5
iopath                    0.1.10
ipython                   8.12.3
ipywidgets                8.1.7
itk                       5.4.3
itk-core                  5.4.3
itk-filtering             5.4.3
itk-io                    5.4.3
itk-numerics              5.4.3
itk-registration          5.4.3
itk-segmentation          5.4.3
itsdangerous              2.2.0
jedi                      0.19.2
Jinja2                    3.1.4
joblib                    1.4.2
jsonschema                4.23.0
jsonschema-specifications 2023.12.1
jupyter_core              5.8.1
jupyterlab_widgets        3.0.15
kiwisolver                1.4.7
lark                      1.2.2
lightning                 2.2.2
lightning-utilities       0.11.9
lit                       15.0.7
markdown-it-py            3.0.0
MarkupSafe                2.1.5
matplotlib                3.7.5
matplotlib-inline         0.1.7
mdurl                     0.1.2
mkl-fft                   1.3.1
mkl-random                1.2.2
mkl-service               2.4.0
mpmath                    1.3.0
multidict                 6.1.0
narwhals                  1.42.1
nbformat                  5.10.4
nest-asyncio              1.6.0
networkx                  3.0
ninja                     1.13.0
nodeenv                   1.9.1
numpy                     1.24.1
nvidia-cublas-cu12        12.1.3.1
nvidia-cuda-cupti-cu12    12.1.105
nvidia-cuda-nvrtc-cu12    12.1.105
nvidia-cuda-runtime-cu12  12.1.105
nvidia-cudnn-cu12         9.1.0.70
nvidia-cufft-cu12         11.0.2.54
nvidia-curand-cu12        10.3.2.106
nvidia-cusolver-cu12      11.4.5.107
nvidia-cusparse-cu12      12.1.0.106
nvidia-nccl-cu12          2.20.5
nvidia-nvjitlink-cu12     12.9.86
nvidia-nvtx-cu12          12.1.105
omegaconf                 2.3.0
open3d                    0.18.0
orjson                    3.10.15
packaging                 24.2
pandas                    2.0.3
parso                     0.8.4
pathtools                 0.1.2
pccm                      0.4.16
peft                      0.13.2
pexpect                   4.9.0
pickleshare               0.7.5
Pillow                    10.0.1
pip                       24.2
pkgutil_resolve_name      1.3.10
platformdirs              4.3.6
plotly                    6.2.0
portalocker               3.0.0
pre-commit                3.5.0
prompt_toolkit            3.0.51
propcache                 0.2.0
protobuf                  4.25.8
psutil                    7.0.0
ptyprocess                0.7.0
pure_eval                 0.2.3
pybind11                  3.0.1
pycparser                 2.21
pydantic                  2.10.6
pydantic_core             2.27.2
pydub                     0.25.1
Pygments                  2.19.2
pyparsing                 3.1.4
pyquaternion              0.9.9
python-dateutil           2.9.0.post0
python-multipart          0.0.20
pytorch-lightning         2.0.0
pytorch3d                 0.7.8     
pytz                      2025.2
PyYAML                    6.0.2
referencing               0.35.1
regex                     2024.11.6
requests                  2.32.4
retrying                  1.4.2
rich                      14.1.0
rpds-py                   0.20.1
ruff                      0.13.0
safehttpx                 0.1.0
safetensors               0.5.3
scikit-learn              1.3.2
scipy                     1.10.1
semantic-version          2.10.0
sentry-sdk                2.33.2
setproctitle              1.3.6
setuptools                75.1.0
shellingham               1.5.4
six                       1.16.0
smmap                     5.0.2
sniffio                   1.3.1
spconv-cu118              2.3.6
stack-data                0.6.3
starlette                 0.44.0
sympy                     1.13.3
termcolor                 2.4.0
threadpoolctl             3.5.0
timm                      1.0.19
tokenizers                0.20.3
tomlkit                   0.12.0
torch                     2.0.1+cu118
torch-cluster             1.6.3+pt20cu118
torch-geometric           2.6.1
torch-scatter             2.1.2+pt20cu118
torchaudio                2.0.2+cu118
torchmetrics              1.5.2
torchvision               0.15.2+cu118
tqdm                      4.67.1
traitlets                 5.14.3
transformers              4.46.3
trimesh                   4.0.2
triton                    2.0.0
typer                     0.17.4
typing_extensions         4.13.2
tzdata                    2025.2
urllib3                   2.2.3
uvicorn                   0.33.0
virtualenv                20.34.0
wandb                     0.15.12
wcwidth                   0.2.13
websockets                12.0
Werkzeug                  3.0.6
wheel                     0.44.0
widgetsnbextension        4.0.14
yarl                      1.15.2
zipp                      3.20.2
